stock_operating_unit/model/stock.py

```diff
@@ -139,6 +139,7 @@ def _check_picking_type_operating_unit(self):
         for rec in self:
             warehouse = rec.picking_type_id.warehouse_id
             if (rec.picking_type_id and rec.operating_unit_id and
+                    rec.picking_type_id.warehouse_id.operating_unit_id and
```

Comment:
Looking at this again, it seems to me that the chech should be like this:

`and warehouse`

 as there is a variable for that and  we are just checking if the warehouse is set in the picking type

Reply:
You are right, changes done